Core/Graphics/Source/GraphicsImpl.h

```diff
@@ -73,6 +74,7 @@ namespace Babylon
         template<typename WindowT>
         WindowT GetNativeWindow();
```

Comment:
Should this be private now since it is only used within this class, and sometimes will be `null`, depending on how you've configured the class?

Reply:
yes, now private.